lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -40,9 +92,9 @@ def install
 
   def clean
     ENV['BUNDLE_GEMFILE'] = @orig_bundle_gemfile
-    FileUtils.remove_entry_secure(@bundler_tmp) if Dir.exist?(@bundler_tmp)
+    @bundler_tmp.rmtree if @bundler_tmp.directory?
 
-    if LanguagePack::Ruby::BUNDLER_VERSION  == "1.7.12"
+    if version  == "1.7.12"
```

Comment:
should we extract this into a constant somewhere?

Reply:
Based on this code I don't think it's possible to ever actually trigger this case. Since we hard code a version and we're not using "1.7.12" it won't ever fire.

Let me know if you agree and I can remove it.